auto_backup/models/db_backup.py

```diff
@@ -301,4 +312,10 @@ def sftp_connection(self):
         else:
             params["password"] = self.sftp_password
 
+        key = paramiko.RSAKey(data=base64.b64decode(self.sftp_hostkey or ''))
+        key_type = self.sftp_hostkey_type
+        cnopts = pysftp.CnOpts()
+        cnopts.hostkeys.add(self.sftp_host, key_type, key)
+        params['cnopts'] = cnopts
```

Comment:
```suggestion
        cnopts = pysftp.CnOpts()
        params['cnopts'] = cnopts
        
        if self.sftp_hostkey:
            key = paramiko.RSAKey(data=base64.b64decode(self.sftp_hostkey))
            key_type = self.sftp_hostkey_type
            cnopts.hostkeys.add(self.sftp_host, key_type, key)
```

Let's make it optional